config/utils.py

```diff
@@ -0,0 +1,19 @@
+# Lint as: python3
+"Config for language-independent sets of graphemes, symbols, etc."
+
+from pynini import *
+
+DEFAULT_INITIAL_PUNCTUATION = union("\"", "'")
```

Comment:
What if we defined [a-z] here as well and imported those into the configs as well, like:
GRAPHEME = union(utils.ALPHA, é, è)

Reply:
I like that plan. Default Latin = [a-z], then in the lang-specific configs we can just add the bonus graphemes that aren't in that set.